pkg/metrics/metrics.go

```diff
@@ -0,0 +1,24 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	PromRegistry = prometheus.NewRegistry()
+
+	// ResourcesInformedCount Counts of informed resources before reconciliation
+	ResourcesInformedCount = promauto.With(PromRegistry).NewCounterVec(prometheus.CounterOpts{
+		Name: "search_collector_resources_informed_count",
+		Help: "Total resources received by informers",
```

Comment:
Let's make this a little more clear. We could receive 100s of events for a single resource, so we are are counting the events, not the resources.
```suggestion
		Name: "search_collector_events_received_count",
		Help: "Total events received by informers",
```

Reply:
Metric renamed